### PR TITLE
[ROX-12565] : Update legacy CVE csv export handler to call CVE type specific handler

### PR DESCRIPTION
**Pull request by charmik-redhat**
## Description

A detailed explanation of the changes in your PR.

Feel free to remove this section if it is overkill for your PR, and the title of your PR is sufficiently descriptive.

## Checklist
- [ ] Investigated and inspected CI test results
- ~[ ] Unit test and regression tests added~
- ~[ ] Evaluated and added CHANGELOG entry if required~
- ~[ ] Determined and documented upgrade steps~
- ~[ ] Documented user facing changes (create PR based on [openshift/openshift-docs](https://github.com/openshift/openshift-docs) and merge into [rhacs-docs](https://github.com/openshift/openshift-docs/tree/rhacs-docs))~

If any of these don't apply, please comment below.

## Testing Performed

Manually tested the CVE CSV endpoint with different CVE Types and compared with the results of calling the type specific endpoints.

In addition to reviewing your code, reviewers **must** also review your testing
instructions and make sure they are sufficient.

### CODE REVIEW

File: central/cve/csv/handler.go
Comment (md2119) [inline central/cve/csv/handler.go:258]: Move this above. The change is not subject to the feature flag. API expects cve type to be provided starting 3.73.0. 

```
- Starting 3.73, CSV export API `/api/vm/export/csv` would require to pass `CVE Type` filter as part of the input query parameter. Requests that do not have the filter would error out.
  - Examples : `CVE Type:NODE_CVE`, `CVE_Type:IMAGE_CVE`, `CVE_TYPE:K8S_CVE`
```

File: central/cve/csv/handler.go
Comment (md2119) [inline central/cve/csv/handler.go:155]: Factor this block out into a func `cveCSVRowsFromLegacyVulnResolver(...)`

```
if !features.PostgresDatastore.Enabled() {
   return cveCSVRowsFromLegacyVulnResolver(...)
}
```

File: central/cve/csv/handler.go
Comment (md2119) [inline central/cve/csv/handler.go:145]: Augment the filename with timestamp- `cve_export_<<ts>>`

**Review by md2119 — APPROVED** [review]
Approved to unblock but address the comments before merging.

File: central/cve/csv/handler.go
Comment (charmik-redhat) [reply central/cve/csv/handler.go:145]: Since adding timestamp is not mentioned in changelog, would it affect the users calling the API through their automated workflows?

**Pull request merged**
